<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogWidget</class>
 <widget class="QWidget" name="LogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>541</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLineEdit" name="edit_name">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>180</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>請輸入email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_pw">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>270</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>請輸入密碼</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_log">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>390</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="text">
      <string>登入</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pw">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>260</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>密碼：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_name">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Email：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_sign">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>390</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="text">
      <string>註冊</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>291</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>網路商店平台 </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_title_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>291</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>註冊帳號</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_pw_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>300</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>密碼：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_email_sign">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>250</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>請輸入email</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_check">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>390</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="text">
      <string>確認</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_name_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Email：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_pw_sign">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>310</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>請輸入密碼</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_back">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>390</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>23</pointsize>
      </font>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pw_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>使用者名稱：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_username">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>180</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>請輸入使用者名稱</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
